viewsets/templates/forms/inline.html

```diff
@@ -0,0 +1,26 @@
+{% load crispy_forms_tags %}
+
+<style>
+    .form-inline .controls {
+        display: inline-block;
+    }
+    .empty-form {
+        display: none;
+    }
+</style>
```

Comment:
I would rather that we don't add styles. We can deal with the base templates not looking perfect. The coder who uses viewsets can decide how to style it